tracing/opentracing/src/main/java/io/helidon/tracing/opentracing/OpenTracingSpan.java

```diff
@@ -91,6 +94,16 @@ public Scope activate() {
         return new OpenTracingScope(tracer.activateSpan(delegate));
     }
 
+    @Override
+    public Span baggage(String key, String value) {
+        return (OpenTracingSpan) delegate.setBaggageItem(key, value);
```

Comment:
This is wrong. The `delegate.setBaggageItem` will return incompatible type.
Also there should be a test to set baggage, which would fail with class cast exception...

Reply:
Now returning this.

Reply:
No you are not returning `this`, but casting delegate to Helidon span, which is the same what you had there before.

Reply:
Yes, when I say "should" it means that there should be a test.